src/api/push.rs

```diff
@@ -152,46 +152,29 @@ pub async fn push_cipher_update(
                 "OrganizationId": (),
                 "RevisionDate": cipher.updated_at
             }
-        });
-
-        send_to_push_relay(data).await;
+        }))
+        .await;
     }
 }
 
-pub async fn push_logout(user: &User, acting_device_uuid: Option<String>, conn: &mut crate::db::DbConn) {
-    if let Some(d) = acting_device_uuid {
-        for device in Device::find_by_user(&user.uuid, conn).await {
-            let data = json!({
-                "userId": user.uuid,
-                "organizationId": (),
-                "deviceId": device.push_uuid,
-                "identifier": d,
-                "type": UpdateType::LogOut as i32,
-                "payload": {
-                    "UserId": user.uuid,
-                    "Date": user.updated_at
-                }
-            });
-            send_to_push_relay(data).await;
+pub async fn push_logout(user: &User, acting_device_uuid: Option<String>) {
```

Comment:
```suggestion
pub fn push_logout(user: &User, acting_device_uuid: Option<String>) {
```
You'll want to avoid automated checks failing again :)

Reply:
Or, enable `pre-commit`, which does all the needed checks so the pipeline will not fail :)

Reply:
Pre-commit? Never heard of it

Reply:
https://pre-commit.com/

I install it via [pipx](https://pypa.github.io/pipx/) `pipx install pre-commit`

Reply:
Yeah, sorry about that, I guess we can squash all the commits on merge so the fix formatting commits aren't cluttering the history. Anyways, should be fixed now.

Reply:
> https://pre-commit.com/
> 
> I install it via [pipx](https://pypa.github.io/pipx/) `pipx install pre-commit`

Thanks for the link, will take a look :)